chart/files/pod-template-file.yaml

```diff
@@ -78,7 +78,7 @@ spec:
     persistentVolumeClaim:
       claimName: {{ template "airflow_dags_volume_claim" . }}
   {{- else if .Values.dags.gitSync.enabled }}
-  - name: dags
+  - name: airflow-dags
```

Comment:
I have a feeling this won't help because the side container uses `dags` name.
See: https://github.com/apache/airflow/blob/3970bfad4c878d99adce80a4bfd824a15132a161/chart/templates/_helpers.yaml#L160